dynamicforms/mixins/single_choice.py

```diff
@@ -0,0 +1,22 @@
+from .render import DisplayMode
+
+
+class SingleChoiceMixin():
+
+    def __init__(self, *args, single_choice_hide=False, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.single_choice_hide = single_choice_hide
```

Comment:
Tu bi bilo morda bolje dat parameter single_choice_display_mode: DisplayMode.

Reply:
Glede na zahteve iz naloge to ni ravno prava pot. Namreč, vseeno je potrebno shranit vrednost, ki bi bila izbrana v selectu. Display mode HIDDEN, skrije select ampak vrednost izbire gre naprej, npr pri SUPPRESS pa pride do težave, ker na submitu tega polja ni in se tudi vrednost ne shrani. Ker polja ni, se tudi mixin ne sproži.

Če nimam prav in lahko kljub vsemu dobim izbrano/privzeto vrednost za shranjevanje mi povej kako.